kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/configuration/NMSettingsConverterTest.java

```diff
@@ -446,6 +447,162 @@ public void buildIpv6SettingsShouldThrowIllegalArgumentExceptionWhenGivenExpecte
         thenIllegalArgumentExceptionThrown();
     }
 
+    @Test
+    public void build8021xSettingsShouldThrowIfIsEmpty() {
+        givenNetworkPropsCreatedWithTheMap(this.internetNetworkPropertiesInstanciationMap);
+
+        whenBuild8021xSettingsIsRunWith(this.networkProperties, "wlan0");
+
+        thenNoSuchElementExceptionThrown();
+    }
+
+    @Test
+    public void build8021xSettingsShouldThrowIfEapIsInvalid() {
+        givenMapWith("net.interface.wlan0.config.802-1x.eap", "invalid eap value");
+        givenNetworkPropsCreatedWithTheMap(this.internetNetworkPropertiesInstanciationMap);
+
+        whenBuild8021xSettingsIsRunWith(this.networkProperties, "wlan0");
+
+        thenIllegalArgumentExceptionThrown();
+    }
+
+    @Test
+    public void build8021xSettingsShouldThrowIfInnerAuthIsInvalid() {
+        givenMapWith("net.interface.wlan0.config.802-1x.eap", "invalid eap value");
+        givenMapWith("net.interface.wlan0.config.802-1x.innerAuth", "invalid Inner Auth value");
+        givenNetworkPropsCreatedWithTheMap(this.internetNetworkPropertiesInstanciationMap);
+
+        whenBuild8021xSettingsIsRunWith(this.networkProperties, "wlan0");
+
+        thenIllegalArgumentExceptionThrown();
+    }
+
+    @Test
+    public void build8021xSettingsTtlsAndMschapV2() {
```

Comment:
Please use the established test naming convention.

```suggestion
    public void build8021xSettingsShouldWorkWithTtlsAndMschapV2() {
```